src/py21cmfast/outputs.py

```diff
@@ -1123,6 +1134,19 @@ def __init__(
         # A *copy* of the current global parameters.
         self.global_params = _globals or dict(global_params.items())
 
+        self.Nion_acg = (
+            mean_f_colls
+            * 10 ** astro_params.F_STAR10
+            * 10 ** astro_params.F_ESC10
+            * self.global_params["Pop2_ion"]
+        )
+        self.Nion_mcg = (
+            mean_f_coll_MINIs
+            * 10 ** astro_params.F_STAR7_MINI
+            * 10 ** astro_params.F_ESC7_MINI
+            * self.global_params["Pop3_ion"]
+        )
+
```

Comment:
Again here, test fails on this part as well (multplying None with something). What do I do with tests then?